Tests/Runtime/NativeAPITest.cs

```diff
@@ -1,112 +1,168 @@
-﻿using System;
+using System;
 using System.Collections;
 using NUnit.Framework;
+using UnityEditor;
 using UnityEngine;
-using Unity.WebRTC;
 using UnityEngine.TestTools;
 
-class NativeAPITest
+namespace Unity.WebRTC.RuntimeTest
 {
-    [AOT.MonoPInvokeCallback(typeof(DelegateDebugLog))]
-    static void DebugLog(string str)
+    class NativeAPITestWithSoftwareEncoder : NativeAPITestWithHardwareEncoder
     {
-        Debug.Log(str);
+        [SetUp]
+        public new void Init()
+        {
+            NativeMethods.RegisterDebugLog(DebugLog);
+            encoderType = EncoderType.Software;
+        }
     }
 
-    [SetUp]
-    public void SetUp()
+    class NativeAPITestWithHardwareEncoder
     {
-        NativeMethods.RegisterDebugLog(DebugLog);
-    }
+        protected EncoderType encoderType;
 
-    [TearDown]
-    public void TearDown()
-    {
-        NativeMethods.RegisterDebugLog(null);
-    }
+        [AOT.MonoPInvokeCallback(typeof(DelegateDebugLog))]
+        protected static void DebugLog(string str)
+        {
+            Debug.Log(str);
+        }
 
-    [Test]
-    public void CreateAndDestroyContext()
-    {
-        var context = NativeMethods.ContextCreate(0);
-        NativeMethods.ContextDestroy(0);
-    }
+        [SetUp]
+        public void Init()
+        {
+            NativeMethods.RegisterDebugLog(DebugLog);
+            encoderType = EncoderType.Software;
```

Comment:
should be Hardware ? The name of the class is NativeAPITestWithHardwareEncoder

Reply:
Fixed.